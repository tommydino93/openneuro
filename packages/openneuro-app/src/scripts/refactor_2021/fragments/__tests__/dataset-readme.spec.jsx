import React from 'react'
import { shallow } from 'enzyme'
import DatasetReadme from '../dataset-readme.jsx'

describe('DatasetReadme component', () => {
  it('renders with common props', () => {
    const wrapper = shallow(<DatasetReadme content="markdown goes here" />)
    expect(wrapper).toMatchSnapshot()
  })
})
