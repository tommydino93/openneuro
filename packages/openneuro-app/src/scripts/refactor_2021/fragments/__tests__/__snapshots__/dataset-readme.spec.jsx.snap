// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatasetReadme component renders with common props 1`] = `
<Fragment>
  <Media
    at="small"
    className=""
  >
    <div
      className="cte-display"
    >
      <div
        className="fade-in"
      >
        <Collapsible
          icon="down-chevron"
          isOpen={false}
          title={
            Array [
              "SHOW MORE",
              "SHOW LESS",
            ]
          }
        >
          <Te>
            markdown goes here
          </Te>
        </Collapsible>
      </div>
    </div>
  </Media>
  <Media
    className=""
    greaterThanOrEqual="medium"
  >
    <div
      className="cte-display"
    >
      <div
        className="fade-in"
      >
        <Te>
          markdown goes here
        </Te>
      </div>
    </div>
  </Media>
</Fragment>
`;
